src/profile-logic/processed-profile-versioning.js

```diff
@@ -327,5 +327,36 @@ const _upgraders = {
       thread.unregisterTime = null;
     }
   },
+  [8]: profile => {
+    // DOMEventMarkerPayload.timeStamp in content process should be in
+    // milliseconds relative to meta.startTime.  Adjust it by adding
+    // the thread.processStartupTime which is the delta to
+    // meta.startTime.
```

Comment:
nit: maybe update the comment to explain why only `timeStamp` is updated here. Maybe `Only the timeStamp property is updated because it's new and perf.html wasn't updated to handle it when it appeared in Firefox.`